packages/larva/lib/utils/getPatternVariants.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Return an array of "variants", or the pattern.XX.js from a 
+ * given directory.
+ * 
+ * @param {string} startPath - the pattern directory to search.
+ */
+
+const fs = require( 'fs' );
+
+// https://regex101.com/r/DgnTR4/2
+const regex = new RegExp( '(.[^.]+)\.(.+)\.js$' );
+
+module.exports = function getPatternVariants( startPath ) {
+
+	// Only return Node prototype files that follow pattern name.XX.js
+	let files = fs.readdirSync( startPath ).filter( ( file ) => {
+		if ( /.+\..*\.js$/.test( file ) ) {
```

Comment:
Seems like this should use the same pattern in `regex` but it was returning a random `k` in the variant list - I think there is something not strict enough about the grouping regex but didn't read up on it enough at the moment